### PR TITLE
Hotfix for facehuggers performance issues

### PR DESCRIPTION
**Pull request by Aranclanos**
### Purpose
Reduces the amount of raycasts that the facehugger search cone does from 20 to 10 and increases the tick rate from 0.3 to 1.

### Notes:
Consider this as a temporal fix for performance issues with facehuggers

### CODE REVIEW

File: UnityProject/Assets/Scripts/NPC/AI/Hostile/FaceHuggerAI.cs [hunk 1]
```diff
@@ -40,7 +40,7 @@ protected override void OnSpawnMob()
 		/// <returns>Gameobject of the first player it found</returns>
 		protected override GameObject SearchForTarget()
 		{
-			var hits = coneOfSight.GetObjectsInSight(hitMask, LayerTypeSelection.Walls , directional.CurrentDirection.Vector, 10f, 20);
+			var hits = coneOfSight.GetObjectsInSight(hitMask, LayerTypeSelection.Walls , directional.CurrentDirection.Vector, 10f, 10);
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/NPC/AI/Hostile/FaceHuggerAI.cs:43]: you could just do circle overlap and then raycast to all the player hits

**Comment by PerfectTangent** [comment]
how much of an improvement do these changes provide for facehuggers?

**Comment by Aranclanos** [comment]
None, its just being called less. They'll take only roughly 15% of what they do atm

**Comment by PerfectTangent** [comment]
> They'll take only roughly 15% of what they do atm

that's what I was looking for.

**Pull request merged**